src/components/quiz/QuizPage.tsx

```diff
@@ -2,18 +2,34 @@ import React, { useState } from "react";
 import { QuizQuestions } from "./QuizQuestions";
 import { UserDetailsForm } from "./UserDetailsForm";
 import { questions } from "./questions";
+import { Reward } from "./Reward";
+import './QuizQuestions.scss';
 
 export const QuizPage: React.FunctionComponent = () => {
   const [isUserLoggedIn, setIsUserLoggedIn] = useState(false);
-
+  const [score,setScore] = useState<number>(0);
+  const [isEndOfQuestion, setIsEndOfQuestion] = useState(false);
+ 
   if (isUserLoggedIn === false) {
```

Comment:
Don't use `===` to test booleans - they already indicate true or false!
```suggestion
  if (!isUserLoggedIn) {
```
I know this wasn't you but I didn't see it last time!